crates/sparrow-runtime/src/prepare/preparer.rs

```diff
@@ -173,6 +168,18 @@ impl Preparer {
             self.time_multiplier,
         )
     }
+    // Prepared files are stored in the following format:
+    // file:///<temp_dir>/<KASKADA_PATH>/tables/<table_uuid>/prepared/<uuid>/part-<n>.parquet
+    pub fn prepared_output_prefix(&self) -> String {
+        let uuid = Uuid::new_v4();
+        let temp_dir = tempfile::tempdir().expect("failed to create temp dir");
```

Comment:
I was thinking about moving this to a `static_init::dynamic` block so we create one temp dir per..sparrow initialization? I wasn't exactly sure when they would be run

Reply:
But, probably more memory-efficient to just create and drop per query since we're not reusing anyway.